src/r/build_package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
We need to decide if this in necessary after all as if you can do everything directly in R and make OS independent would be better.

Reply:
Yes. I think that's the right way to go. All the package development things can be done in R.

Reply:
OK so we need to update `DEVELOP.md` to add a command that copy the license. Then we can remove this and update the CI accordingly.

Reply:
Cool. I will update `DEVELOP.md` accordingly.